internal/controller/imagerepository_controller.go

```diff
@@ -48,11 +48,15 @@ const (
 
 	ImageRepositoryFinalizer = "appstudio.openshift.io/image-repository"
 
-	buildPipelineServiceAccountName      = "appstudio-pipeline"
-	updateComponentAnnotationName        = "image-controller.appstudio.redhat.com/update-component-image"
-	additionalUsersConfigMapName         = "image-controller-additional-users"
-	additionalUsersConfigMapKey          = "quay.io"
-	skipRepositoryDeletionAnnotationName = "image-controller.appstudio.redhat.com/skip-repository-deletion"
+	buildPipelineServiceAccountName               = "appstudio-pipeline"
+	updateComponentAnnotationName                 = "image-controller.appstudio.redhat.com/update-component-image"
+	additionalUsersConfigMapName                  = "image-controller-additional-users"
+	additionalUsersConfigMapKey                   = "quay.io"
+	skipRepositoryDeletionAnnotationName          = "image-controller.appstudio.redhat.com/skip-repository-deletion"
+	waitForRelatedComponentInitialDelay           = 5
```

Comment:
nitpick: we can separate delay constants by an empty line

Reply:
done